spec/commands/config_spec.rb

```diff
@@ -102,6 +136,12 @@
       run "puts Bundler.settings['local.foo']"
       expect(out).to eq(File.expand_path(Dir.pwd + "/.."))
     end
+
+    it "saves with parseable option" do
+      bundle "config --global --parseable foo value"
```

Comment:
(i.e. it's `You are replacing the current local value of foo, which is currently nil`)